src/hooks/auth0-provider.tsx

```diff
@@ -40,13 +41,21 @@ const getIdTokenProfileClaims = idToken => {
  *   <App />
  * </Auth0Provider>
  */
-const Auth0Provider = ({domain, clientId, children}) => {
+const Auth0Provider = ({
+  domain,
+  clientId,
+  children,
+}: {
+  domain: string;
+  clientId: string;
+  children: any;
+}) => {
```

Comment:
We should be able to use `PropsWithChildren<{ domain: string; clientId: string;}>` here.